core/sql/sqlcomp/CmpSeabaseDDLupgrade.cpp

```diff
@@ -314,7 +326,7 @@ short CmpSeabaseMDupgrade::executeSeabaseMDupgrade(CmpDDLwithStatusInfo *mdui,
   // now as they are stateless
   CmpSeabaseUpgradeRepository upgradeRepository;
   CmpSeabaseUpgradePrivMgr upgradePrivMgr;
-
+  CmpSeabaseUpgradeLibraries upgradeLibraries;
```

Comment:
Since we create repository, libraries, and then privmgr, perhaps upgradeLibraries should come before upgradePrivMgr?